app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
+            shopify_refund = ShopifyAPI::Refund.find(shopify_refund_id, params: { order_id: shopify_order_id })
+            Spree::Retail::Shopify::GenerateRefundOrder.new(shopify_refund).process
+
+            head :ok
+          end
+
+          def shopify_refund_id
+            params[:id]
+          end
+
+          def shopify_order_id
+            params[:order_id]
```

Comment:
If the response code is not 200 then Shopify re-sends the event.

I don't see situations here the `order_id` params wouldn't be set on the Shopify side of it. What would you suggest here?